src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf?) format("truetype"),
+    url(./fonts/Avenir-Heavy.ttf) format("ttf"),
+    url(./fonts/Avenir.ttc) format("ttc"),;
+  font-weight: 300;
```

Comment:
you need to create a separate font-face for each font and font weight
Avenir-Heavy.ttf is definitely not `font-weight: 300`